tests/tests_e3/fs/main_test.py

```diff
@@ -191,6 +191,7 @@ def test_tree_state():
     time.sleep(2)
 
     e3.os.fs.touch("toto")
+    e3.os.fs.touch("toto2")
```

Comment:
The `touch` is not necessary here. You never test "toto2" before writing something to it. Since you open it by writing, the file will be created anyway if it doesn't exist.

Reply:
No, here, I create toto2 before to check that we are generating another hash with content of 2 files, this hash will be compared with the first one that don't look at the content (state4 vs state9